esdl/cube.py

```diff
@@ -75,8 +78,15 @@ def open(base_dir):
 
         if not os.path.exists(base_dir):
             raise IOError('data cube base directory does not exists: %s' % base_dir)
-        config = CubeConfig.load(os.path.join(base_dir, 'cube.config'))
-        return Cube(base_dir, config)
+        zg = zarr.open_group(base_dir)
```

Comment:
Please leave a code comment to indicate what's going on in the following. Or better extract a private method with a name indicating what it does and a comment why it is needed.

Reply:
I have re-written this part so that cube attributes are written through xarray and not through pure zarr, which should improve readability of this part